src/sentry/seer/similarity/similar_issues.py

```diff
@@ -91,7 +87,14 @@ def get_similarity_data_from_seer(
                 },
             )
         except SimilarGroupNotFoundError:
-            metrics.incr("seer.similar_issue_request.parent_issue", tags={"outcome": "not_found"})
```

Comment:
For reference, how often does this occur in practice?

Reply:
> Once @jangjodi's deletion work is done, though, and Seer records get deleted as soon as their corresponding GroupHash records disappear, it shouldn't ever happen 

Very soon, theoretically never.